dbclient/HiveClient.py

```diff
@@ -387,6 +400,7 @@ def import_hive_metastore(self, cluster_name=None, metastore_dir='metastore/', v
                     print("Importing view {0}.{1}".format(db_name, view_name))
                     local_view_ddl = metastore_view_dir + db_name + '/' + view_name
                     is_successful = self.apply_table_ddl(local_view_ddl, ec_id, cid, db_path, has_unicode)
+                    checkpoint_metastore_set.write(full_table_name)
```

Comment:
This operation has been checkpointed but the checkpoint is not used?

Reply:
I thought it would be handled by the checkpoint check in the forloop block before this as the views are only moved from there. But there would be scenario where we want to check for checkpoint here too. Good catch, added that